RELEASES.md

```diff
@@ -6,7 +6,7 @@ Throughout this document, replace $OLD_VERSION with the previous release and $NE
 
 1. Update your local master branch
 2. Review the changes in git since the last release with a command like `git log --graph $OLD_VERSION...master` or online at https://github.com/gravitystorm/openstreetmap-carto/compare/$OLD_VERSION...master
-3. Review [pull requests](https://github.com/gravitystorm/openstreetmap-carto/pulls) for anything that should be merged before release
+3. Review [pull requests](pulls) for anything that should be merged before release
```

Comment:
This won't work everywhere the file can be viewed. For example, on `https://github.com/gravitystorm/openstreetmap-carto/blob/47ad08b8cc89a0afb181e984cd843a626536a982/RELEASES.md` the link directs to `https://github.com/gravitystorm/openstreetmap-carto/blob/47ad08b8cc89a0afb181e984cd843a626536a982/pulls`

The same also applies to the 711 issue link.

Reply:
@pnorman Do we have to use the full link (and is it actually needed in practice), or can we use a link operand like `./` to satify that need? See https://github.com/gravitystorm/openstreetmap-carto/pull/4570#discussion_r891711823 if you haven't yet, though I'm sure you are aware of GitHub's recommendations.
(Obviously, I'm fine with a full link, I don't want to [make it too complicated](https://github.com/github/markup/issues/1502).)

Reply:
Pretty sure any links that aren't to other files will need to be proper HTTP URLs